common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -89,7 +89,7 @@ def test_bulk_email_a11y(self):
         ])
         self.send_email_page.a11y_audit.config.set_rules({
             "ignore": [
-                'button-name',  # TODO: AC-491
+                'button-name',  # TODO: TNL-5830
```

Comment:
This is vendor file issue. I have linked it to our task about the appropriate vendor.